packages/rest-typings/src/v1/rooms.ts

```diff
@@ -423,6 +423,40 @@ const GETRoomsNameExistsSchema = {
 
 export const isGETRoomsNameExists = ajv.compile<GETRoomsNameExists>(GETRoomsNameExistsSchema);
 
+type RoomsIsUserMuted = { userId: string; roomId: string } | { username: string; roomId: string };
+
+const RoomsIsUserMutedSchema = {
+	type: 'object',
+	oneOf: [
+		{
+			properties: {
+				userId: {
+					type: 'string',
+				},
+				roomId: {
+					type: 'string',
+				},
```

Comment:
```suggestion
				userId: {
					type: 'string',
					minLength: 1,
				},
				roomId: {
					type: 'string',
					minLength: 1,
				},
```
I think it'd be good not to accept empty strings. We handle this case internally, but since ajv can check it, then why not? (this also avoids calling `Rooms.findOne` searching for an empty `_id`, which returns `undefined` for us but I think that's something to avoid)
Sorry for not suggesting this before, though

Reply:
Nice idea, don't worry about it. :)